owltorrent-core/CourseraDSFundamentals.torrent

```diff
@@ -0,0 +1,132 @@
-and-basic-sql12:02_basic-sql27:03_count-distinct-limit.mp4eed4:attr1:p6:lengthi72376e4:pathl4:.pad3:104eed6:lengthi3519648e4:pathl62:[TutsNode.com] - Data Science Fundamentals with Python and SQL34:python-for-applied-data-science-ai25:02_python-data-structures13:02_dictionary19:01_dictionaries.mp4eed4:attr1:p6:lengthi150368e4:pathl4:.pad3:105eed6:lengthi3456423e4:pathl62:[TutsNode.com] - Data Science Fundamentals with Python and SQL34:statistics-for-data-science-python22:06_regression-analysis44:01_regression-in-place-of-hypothesis-testing36:02_regression-in-place-of-t-test.mp4eed4:attr1:p6:lengthi213593e4:pathl4:.pad3:106eed6:lengthi3287571e4:pathl62:[TutsNode.com] - Data Science Fundamentals with Python and SQL16:sql-data-science49:01_week-1-introduction-to-databases-and-basic-sql28:01_introduction-to-databases38:01_welcome-to-sql-for-data-science.mp4eed4:attr1:p6:lengthi382445e4:pathl4:.pad3:107eed6:lengthi3280911e4:pathl62:[TutsNode.com] - Data Science Fundamentals with Python and SQL34:statistics-for-data-science-python22:06_regression-analysis44:01_regression-in-place-of-hypothesis-testing41:04_regression-in-place-of-correlation.mp4eed4:attr1:p6:lengthi389105e4:pathl4:.pad3:108eed6:lengthi3250965e4:pathl62:[TutsNode.com] - Data Science Fundamentals with Python and SQL34:statistics-for-data-science-python40:01_course-introduction-and-python-basics40:01_course-introduction-and-python-basics39:03_python-packages-for-data-science.mp4eed4:attr1:p6:lengthi419051e4:pathl4:.pad3:109eed6:lengthi3151995e4:pathl62:[TutsNode.com] - Data Science Fundamentals with Python and SQL34:open-source-tools-for-data-science26:01_data-scientists-toolkit28:01_languages-of-data-science32:02_languages-of-data-science.mp4eed4:attr1:p6:lengthi518021e4:pathl4:.pad3:110eed6:lengthi3030615e4:pathl62:[TutsNode.com] - Data Science Fundamentals with Python and SQL16:sql-data-science42:03_week-3-accessing-databases-using-python35:01_accessing-databases-using-python48:03_connecting-to-a-database-using-ibm-db-api.mp4eed4:attr1:p6:lengthi115113e4:pathl4:.pad3:111eed6:lengthi2748487e4:pathl62:[TutsNode.com] - Data Science Fundamentals with Python and SQL34:python-for-applied-data-science-ai30:04_working-with-data-in-python9:03_pandas42:02_pandas-working-with-and-saving-data.mp4eed4:attr1:p6:lengthi397241e4:pathl4:.pad3:112eed6:lengthi1490893e4:pathl62:[TutsNode.com] - Data Science Fundamentals with Python and SQL34:open-source-tools-for-data-science29:03_ibm-tools-for-data-science18:02_other-ibm-tools39:01_other-ibm-tools-for-data-science.mp4eed4:attr1:p6:lengthi81971e4:pathl4:.pad3:113eed6:lengthi1107304e4:pathl62:[TutsNode.com] - Data Science Fundamentals with Python and SQL34:statistics-for-data-science-python44:04_introduction-to-probability-distributions28:01_probability-distributions42:07_standard-normal-table_instructions.htmleee4:name45:Data Science Fundamentals with Python and SQL12:piece lengthi524288e6:pieces30840:V	���=�7�Њb�suF"�6E�q�/{5�`�k\&3��+�^�ˠ� ��U���
```

Comment:
I'd prefer not adding another duplicate, but I guess it's fine. (You should be able to mingle the path with "../"s to navigate to the root. )

Reply:
Leaving it in this package for now